worlds/aquaria/docs/en_Aquaria.md

```diff
@@ -0,0 +1,61 @@
+# Aquaria
+
+## Where is the settings page?
+
+The player settings page for this game contains all the options you need to configure and export a config file. Player
+settings page link: [Aquaria Player Settings Page](../player-settings).
```

Comment:
```suggestion
## Where is the options page?

The player options page for this game contains all the options you need to configure and export a config file. Player
optionss page link: [Aquaria Player Options Page](../player-options).
```
As of 0.4.6, all player configurations are called options. `../player-settings` now leads to a 404